Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/DriverLocation.hs

```diff
@@ -106,12 +107,12 @@ updateOnRide driverId onRide merchantId = do
         mDriverLocatation
   CDI.updateOnRide driverId onRide
 
-updateOnRideCache :: (CacheFlow m r, Esq.EsqDBFlow m r) => Id Person.Driver -> m ()
+updateOnRideCache :: (CacheFlow m r, Esq.EsqDBFlow m r, EsqLocDBFlow m r) => Id Person.Driver -> m ()
 updateOnRideCache driverId = do
   driverLocation <- DLQueries.findById (cast driverId) >>= fromMaybeM (PersonNotFound driverId.getId)
   cacheDriverLocation driverLocation
 
-updateOnRideCacheForCancelledOrEndRide :: (CacheFlow m r, EsqDBReplicaFlow m r, EsqDBFlow m r) => Id Person.Driver -> Id Merchant -> m ()
+updateOnRideCacheForCancelledOrEndRide :: (CacheFlow m r, EsqDBReplicaFlow m r, EsqDBFlow m r, EsqLocDBFlow m r) => Id Person.Driver -> Id Merchant -> m ()
 updateOnRideCacheForCancelledOrEndRide driverId merchantId = do
   mbDriverLocation <- findById merchantId (cast driverId)
   maybe
```

Comment:
`DLQueries.upsertGpsCoord` needs to run in location DB?

Reply:
Done